exercises/variable-length-quantity/tests/variable-length-quantity.rs

```diff
@@ -0,0 +1,89 @@
+extern crate variable_length_quantity as vlq;
+
+#[test]
+fn to_single_byte() {
+    assert_eq!(&[0x40], vlq::to_bytes(&[0x40]).as_slice());
+    assert_eq!(&[0x7f], vlq::to_bytes(&[0x7f]).as_slice());
+}
+
+#[test]
+#[ignore]
+fn to_single_byte_zero() {
+    assert_eq!(&[0x00], vlq::to_bytes(&[0x00]).as_slice());
```

Comment:
Should this test go first? It seems like it's the simplest to pass, and I usually put the simplest-to-pass tests first.